Classes/Common/MetsDocument.php

```diff
@@ -245,6 +281,24 @@ public function getFileLocation(string $id): string
     }
 
     /**
+     * {@inheritDoc}
```

Comment:
You can't use `{@inheritDoc}` for a method which does not exist in parent class.